README.md

```diff
@@ -17,9 +17,9 @@ Conjur provider for [Summon](https://github.com/cyberark/summon).
 Pre-built binaries and packages are available from GitHub releases
 [here](https://github.com/cyberark/summon-conjur/releases).
 
-### Using summon-conjur with Conjur OSS 
+### Using summon-conjur with Conjur Open Source 
 
-Are you using this project with [Conjur OSS](https://github.com/cyberark/conjur)? Then we 
+Are you using this project with [Conjur Open Source](https://github.com/cyberark/conjur)? Then we 
 **strongly** recommend choosing the version of this project to use from the latest [Conjur OSS 
```

Comment:
Should this one be changed also?

Reply:
No - it runs onto the next line as `Conjur OSS suite`, which is still a vali. It's one of the only phrases where `Conjur OSS` is still accepted.